pytorch/pytorch-engine/README.md

```diff
@@ -1,195 +1,195 @@
-    <version>1.7.0</version>
-    <scope>runtime</scope>
-</dependency>
-```
-
-### Windows
-
-For the Windows platform, you can choose between CPU and GPU.
-
-#### Windows GPU
-
-- ai.djl.pytorch:pytorch-native-cu110:1.7.0:win-x86_64
-- ai.djl.pytorch:pytorch-native-cu102:1.7.0:win-x86_64
-- ai.djl.pytorch:pytorch-native-cu101:1.7.0:win-x86_64
-
-```xml
-<dependency>
-    <groupId>ai.djl.pytorch</groupId>
-    <artifactId>pytorch-native-cu102</artifactId>
-    <classifier>win-x86_64</classifier>
-    <version>1.7.0</version>
-    <scope>runtime</scope>
-</dependency>
-```
-
-```xml
-<dependency>
-    <groupId>ai.djl.pytorch</groupId>
-    <artifactId>pytorch-native-cu101</artifactId>
-    <classifier>win-x86_64</classifier>
-    <version>1.7.0</version>
-    <scope>runtime</scope>
-</dependency>
-```
-
-### Windows CPU
-
-- ai.djl.pytorch:pytorch-native-cpu:1.7.0:win-x86_64
-
-```xml
-<dependency>
-    <groupId>ai.djl.pytorch</groupId>
-    <artifactId>pytorch-native-cpu</artifactId>
-    <classifier>win-x86_64</classifier>
-    <scope>runtime</scope>
-    <version>1.7.0</version>
-</dependency>
-```
+# DJL - PyTorch engine implementation
+
+## Overview
+This module contains the Deep Java Library (DJL) EngineProvider for PyTorch.
+
+We don't recommend that developers use classes in this module directly.
+Use of these classes will couple your code with PyTorch and make switching between frameworks difficult.
+
+## Documentation
+
+The latest javadocs can be found on the [djl.ai website](https://javadoc.io/doc/ai.djl.pytorch/pytorch-engine/latest/index.html).
+
+You can also build the latest javadocs locally using the following command:
+
+```sh
+# for Linux/macOS:
+./gradlew javadoc
+
+# for Windows:
+..\..\gradlew javadoc
+```
+The javadocs output is built in the `build/doc/javadoc` folder.
+
+## Installation
+You can pull the PyTorch engine from the central Maven repository by including the following dependency:
+
+- ai.djl.pytorch:pytorch-engine:0.8.0
+
+```xml
+<dependency>
+    <groupId>ai.djl.pytorch</groupId>
+    <artifactId>pytorch-engine</artifactId>
+    <version>0.8.0</version>
+    <scope>runtime</scope>
+</dependency>
+```
+Besides the `pytorch-engine` library, you may also need to include the PyTorch native library in your project.
+All current provided PyTorch native libraries are downloaded from [PyTorch C++ distribution](https://pytorch.org/get-started/locally/#start-locally).
+
+Choose a native library based on your platform and needs:
+
+### Automatic (Recommended)
+
+We offer an automatic option that will download the native libraries into [cache folder](../../docs/development/cache_management.md) the first time you run DJL.
+It will automatically determine the appropriate jars for your system based on the platform and GPU support.
+
+- ai.djl.pytorch:pytorch-native-auto:1.6.0
+
+```xml
+<dependency>
+    <groupId>ai.djl.pytorch</groupId>
+    <artifactId>pytorch-native-auto</artifactId>
+    <version>1.6.0</version>

```

Comment:
we are going to release 0.9.0 which works with pytorch 1.7.0 after Thanksgiving. Meanwhile, you can leverage 0.9.0-SNAPSHOT to try pytorch 1.7.0

Reply:
This yeah can actually not update as long as user knows they need to use snapshot or the lower version.Just a side edit.Can be scraped.